src/raftstore/store/worker/apply.rs

```diff
@@ -2165,6 +2206,7 @@ impl Runner {
         APPLY_PROPOSAL.observe(propose_num as f64);
     }
 
+    /// Handles peer registration. When a peer is created, it will register a apply delegate.
```

Comment:
```suggestion
    /// Handles peer registration. When a peer is created, it will register an apply delegate.
```

When you have `... a a....` or `... a e...` it's good to use `an` over `a`. Otherwise when you read it out, it sounds odd! "I have a apple" vs "I have an apple!"

Reply:
Oh, thanks!